src/Symfony/Component/Cache/Tests/Adapter/DoctrineDbalAdapterTest.php

```diff
@@ -48,6 +48,10 @@ public function createCachePool(int $defaultLifetime = 0): CacheItemPoolInterfac
 
     public function testConfigureSchemaDecoratedDbalDriver()
     {
+        if (file_exists(self::$dbFile)) {
+            @unlink(self::$dbFile);
+        }
```

Comment:
Why not do this in the setup or teardown?

Reply:
Some of the tests coming from the parent class rely on the schema to exist.